{
object_name_1: "box1",

object_name_2: "box1_3",

object_name_3: "box1_4",

object_name_4: "cube",

object_name_5: "box1",

object_name_6: "box2",

object_name_7: "box3",

object_name_8: "box3",

n_object: 3
}